spec/services/resolver_service_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+describe ResolverService do
+  let(:user) { build_stubbed(:user) }
+
+  subject(:resolver) { described_class.new(object, user) }
+
+  describe '#process' do
+    Timecop.freeze(Time.zone.now) do
+      before do
+        resolver.process
+      end
+      context 'when created with an application' do
```

Comment:
this could be just a single line:

```ruby
before { resolver.process }
```

plus a white space could be added :neckbeard: